superset-frontend/packages/generator-superset/generators/plugin-chart/templates/README.erb

```diff
@@ -1,52 +1,40 @@
-## superset-plugin-chart-<%= packageName %>
+# <%= packageName %>
 
-This plugin provides <%= description %> for Superset.
+This is the <%= description %> Superset Chart Plugin.
 
 ### Usage
 
-Configure `key`, which can be any `string`, and register the plugin. This `key` will be used to lookup this chart throughout the app.
+To build the plugin, run the following commands:
 
-```js
-import <%= packageLabel %>ChartPlugin from '@superset-ui/plugin-chart-<%= packageName %>';
+```
+npm i --force
```

Comment:
this is unfortunately needed due to the conflicting versions we currently have in our deps (React, AntD etc) 😏

Reply:
We can also use `--legacy-peer-deps`, looks a bit less aggressive 😛

Reply:
With `--legacy-peer-deps`, the React and the core packages won't be present due to them being peer dependencies, so running tests will fail:

```bash
npm run test
```

```
src/types.ts:19:68 - error TS2307: Cannot find module '@superset-ui/core' or its corresponding type declarations.

19 import { QueryFormData, supersetTheme, TimeseriesDataRecord } from '@superset-ui/core';
                                                                      ~~~~~~~~~~~~~~~~~~~

src/SupersetPluginChartHelloWorld.tsx:19:45 - error TS2307: Cannot find module 'react' or its corresponding type declarations.

19 import React, { useEffect, createRef } from 'react';
                                               ~~~~~~~
```